### PR TITLE
grid assignment

### PR DESCRIPTION
**Pull request by HawkeyeUW**
Grid printer exercise in Lesson 2.  Parts 1,2,3

### CODE REVIEW

**Comment by HawkeyeUW** [comment]
Added fizzbuzz and series code to repo.

File: students/jvirtue/lesson02/fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    if x == 0:
+        print(0)
+    elif x%3 == 0 and x%5 == 0:
+        print("FizzBuz")
+    elif x%3 == 0:
+        print("Fizz")
+    elif x%5 == 0:
+        print("Buzz")
+    else:
+        print(x)
```
Comment (charlesrobison) [inline students/jvirtue/lesson02/fizzbuzz.py]: Looks good - but only goes to 99.  Should go to 100.

File: students/jvirtue/lesson02/grid.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+print()
+print(vertical,end="|")
+print()
+print(vertical,end="|")
+print()
+print(horizontal,end="+") """
+
+#Print a grid based on parameter Part II
+#One parameter to control size of grid
+""" def printgrid(bars):
```
Comment (charlesrobison) [inline students/jvirtue/lesson02/grid.py]: Why the comment block?

For function names, they should be snake_case in Python per PEP8

File: students/jvirtue/lesson02/grid.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+#Two parameters to control rows,columns,unit size
+def printgrid(count,bars):
+    for y in range(count):
+        print(("+"+"-"*bars)*count,end="+")
+        print()
+        for x in range(bars):
+            print(("|"+" "*bars)*count,end="|")
+            print()
+    print(("+"+"-"*bars)*count,end="+")
+    return;
```
Comment (charlesrobison) [inline students/jvirtue/lesson02/grid.py]: Semi colon not necessary.

File: students/jvirtue/lesson02/series.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+assert lucas(7) == 29
+
+#Test out sum_series to see if it works
+assert sum_series(5) == 5
+assert sum_series(7) == 13
+
+#Test out sum_series with optional parameters
+assert sum_series(5,2,1) == 11
+assert sum_series(7,2,1) == 29
+
```
Comment (charlesrobison) [inline students/jvirtue/lesson02/series.py]: Looks good - nice assert statements!

File: students/jvirtue/lesson02/fizzbuzz.py (hunk 1 above)
Comment (HawkeyeUW) [reply students/jvirtue/lesson02/fizzbuzz.py]: Made updates to this pull request https://github.com/UWPCE-PythonCert-ClassRepos/Self_Paced-Online/pull/1347

File: students/jvirtue/lesson02/grid.py (hunk 2 above)
Comment (HawkeyeUW) [reply students/jvirtue/lesson02/grid.py]: Made updates to this pull request #1347

File: students/jvirtue/lesson02/grid.py (hunk 3 above)
Comment (HawkeyeUW) [reply students/jvirtue/lesson02/grid.py]: Made updates to this pull request #1347

**Pull request merged**